extension/src/popup/components/mnemonicPhrase/CheckButton.tsx

```diff
@@ -1,30 +1,30 @@
 import React from "react";
 import styled from "styled-components";
 import { Field } from "formik";
-import { COLOR_PALETTE } from "popup/constants/styles";
 
 type CheckButtonProps = {
   onChange: (e: any) => void;
   wordKey: string;
   word: string;
 };
 const ButtonLabelEl = styled.label`
```

Comment:
Should we move these styled-component styles to a CSS file?

Reply:
yes good point https://github.com/stellar/freighter/pull/359/commits/1e9f8c95ac98a3568b3f2e8caadbd8a9b1002856